tests/test_part_image.py

```diff
@@ -31,16 +31,20 @@ def align_up(bytes, alignment):
 def extract_partition(img, number):
     output = subprocess.Popen(["fdisk", "-l", "-o", "device,start,end", img],
                               stdout=subprocess.PIPE)
+    start = None
+    end = None
     for line in output.stdout:
         if re.search("img%d" % number, line.decode()) is None:
             continue
 
-        match = re.match("\s*\S+\s+(\S+)\s+(\S+)", line.decode())
+        match = re.match(r"\s*\S+\s+(\S+)\s+(\S+)", line.decode())
```

Comment:
Nice. I've been also thinking about fixing bunch of those warnings. There are still quite a few I think.

Reply:
:+1: